package.json

```diff
@@ -101,19 +102,20 @@
     "build": "yarn run clean && yarn run build-rb && yarn run lint && yarn run dev",
     "build-ci": "./node_modules/.bin/webpack --progress --colors --config build/webpack.config.js",
     "build-rb": "mojito-rb-gen -s src/i18n -o src/i18n/json -b en-US.properties",
-    "ci": "yarn run clean && yarn run build-rb && yarn run lint && yarn run test && yarn run build-ci",
+    "ci": "yarn install && yarn nsp && yarn run clean && yarn run build-rb && yarn run lint && yarn run test && yarn run build-ci",
     "clean": "rm -rf dist && rm -rf reports/coverage && rm -rf src/i18n/json",
     "commitmsg": "commitlint -e",
     "debug": "yarn install && NODE_ENV=test ./node_modules/.bin/karma start build/karma.conf.js --no-single-run --auto-watch",
     "dev": "yarn install && BABEL_ENV=dev NODE_ENV=dev ./node_modules/.bin/webpack --progress --colors --config build/webpack.config.js",
     "functional-tests": "python -m SimpleHTTPServer 8000 & node ./node_modules/codeceptjs/bin/codecept.js run --verbose ; killall python -m SimpleHTTPServer 8000",
     "functional-tests-ci": "yarn run clean && yarn run build-rb && yarn run build-ci && node ./node_modules/codeceptjs/bin/codecept.js run --verbose --grep @ci",
     "lint": "NODE_ENV=dev ./node_modules/.bin/eslint src/lib && ./node_modules/.bin/stylelint 'src/lib/**/*.scss'",
+    "nsp": "nsp check --reporter summary",
     "precommit": "lint-staged",
     "prepush": "yarn run lint",
     "prettier": "prettier-eslint \"src/lib/**/*.js\" --print-width 120 --single-quote --tab-width 4 --write",
     "prod": "yarn run build-rb && BABEL_ENV=production NODE_ENV=production node --max_old_space_size=4096 ./node_modules/webpack/bin/webpack.js --progress --colors --config build/webpack.config.js",
-    "release": "yarn run clean && yarn run build-rb && yarn run lint && yarn run test && yarn run prod",
+    "release": "yarn install && yarn nsp && yarn run clean && yarn run build-rb && yarn run lint && yarn run test && yarn run prod",
```

Comment:
does this make sense to only do in the jenkins release or when we're in the process of cutting the release/build locally?